app/src/main/res/drawable/progress_background_circle.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <size
+        android:width="56dp"
+        android:height="56dp" />
+    <solid android:color="#444444" />
```

Comment:
Please add color in the `color.xml` and `color.xml(night)` this will automatically pick right color for the required day/night theme.